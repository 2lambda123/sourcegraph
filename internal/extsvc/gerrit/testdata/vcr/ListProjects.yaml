---
version: 1
interactions:
- request:
    body: ""
    form: {}
    headers: {}
    url: https://gerrit.sgdev.org/a/projects/?S=0&n=5
    method: GET
  response:
    body: |
      )]}'
      {
        "All-Projects": {
          "id": "All-Projects",
          "state": "ACTIVE"
        },
        "All-Users": {
          "id": "All-Users",
          "state": "ACTIVE"
        },
        "idan-temp": {
          "id": "idan-temp",
          "state": "ACTIVE"
        },
        "k8s-gerrit": {
          "id": "k8s-gerrit",
          "state": "ACTIVE"
        },
        "sourcegraph": {
          "id": "sourcegraph",
          "state": "ACTIVE"
        }
      }
    headers:
      Cache-Control:
      - no-cache, no-store, max-age=0, must-revalidate
      Cf-Cache-Status:
      - DYNAMIC
      Cf-Ray:
      - 72125428b96e2a96-ORD
      Content-Disposition:
      - attachment
      Content-Type:
      - application/json;charset=utf-8
      Date:
      - Sun, 26 Jun 2022 01:56:50 GMT
      Expect-Ct:
      - max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
      Expires:
      - Mon, 01 Jan 1990 00:00:00 GMT
      Pragma:
      - no-cache
      Server:
      - cloudflare
      Strict-Transport-Security:
      - max-age=15724800; includeSubDomains
      X-Content-Type-Options:
      - nosniff
      X-Frame-Options:
      - DENY
    status: 200 OK
    code: 200
    duration: ""
- request:
    body: ""
    form: {}
    headers: {}
    url: https://gerrit.sgdev.org/a/projects/?S=0&n=5&type=CODE
    method: GET
  response:
    body: |
      )]}'
      {
        "idan-temp": {
          "id": "idan-temp",
          "state": "ACTIVE"
        },
        "k8s-gerrit": {
          "id": "k8s-gerrit",
          "state": "ACTIVE"
        },
        "sourcegraph": {
          "id": "sourcegraph",
          "state": "ACTIVE"
        }
      }
    headers:
      Cache-Control:
      - no-cache, no-store, max-age=0, must-revalidate
      Cf-Cache-Status:
      - DYNAMIC
      Cf-Ray:
      - 721254297a6a2a96-ORD
      Content-Disposition:
      - attachment
      Content-Type:
      - application/json;charset=utf-8
      Date:
      - Sun, 26 Jun 2022 01:56:50 GMT
      Expect-Ct:
      - max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
      Expires:
      - Mon, 01 Jan 1990 00:00:00 GMT
      Pragma:
      - no-cache
      Server:
      - cloudflare
      Strict-Transport-Security:
      - max-age=15724800; includeSubDomains
      X-Content-Type-Options:
      - nosniff
      X-Frame-Options:
      - DENY
    status: 200 OK
    code: 200
    duration: ""
