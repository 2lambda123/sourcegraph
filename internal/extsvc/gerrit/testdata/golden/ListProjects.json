{
  "idan-temp": {
   "description": "",
   "id": "idan-temp",
   "name": "",
   "parent": "",
   "state": "ACTIVE",
   "branches": null,
   "labels": null
  },
  "k8s-gerrit": {
   "description": "",
   "id": "k8s-gerrit",
   "name": "",
   "parent": "",
   "state": "ACTIVE",
   "branches": null,
   "labels": null
  },
  "sourcegraph": {
   "description": "",
   "id": "sourcegraph",
   "name": "",
   "parent": "",
   "state": "ACTIVE",
   "branches": null,
   "labels": null
  }
 }